controllers/idler/idler_controller_test.go

```diff
@@ -627,3 +638,7 @@ func prepareReconcileWithPodsRunningTooLong(t *testing.T, idler toolchainv1alpha
 	memberoperatortest.AssertThatIdler(t, idler.Name, cl).TracksPods(payloads.allPods)
 	return reconciler, req, cl, allCl
 }
+
+func newGetHostClusterReady(fakeClient client.Client) cluster.GetHostClusterFunc {
```

Comment:
`getHostCluster` is simpler to read than `newGetHostClusterReady`
```suggestion
func getHostCluster(fakeClient client.Client) cluster.GetHostClusterFunc {
```

Reply:
by the way, you could probably also test the case where the returned func will return an error when trying to the Host cluster

Reply:
renamed in 7ccbf5d